### PR TITLE
1st step to move on ITK5.1.1

### PR DESCRIPTION
**Pull request by Florent2305**
1st step to move on ITK5.1.1

### CODE REVIEW

**Comment by nebatmusic** [comment]
good.

**Review by mathildemerle — APPROVED** [review]

File: src/plugins/legacy/itkFilters/CMakeLists.txt [hunk 1]
```diff
@@ -73,8 +73,8 @@ add_library(${TARGET_NAME} SHARED
 target_link_libraries(${TARGET_NAME}
   ${QT_LIBRARIES}
   dtkCore
-  dtkLog
-  ITKCommon
+  dtkLog  
+  ${ITK_LIBRARIES}
```
Comment (ocommowi) [inline src/plugins/legacy/itkFilters/CMakeLists.txt:77]: Ugly, can't we use only the right libs ?

File: src/plugins/legacy/medSegmentation/CMakeLists.txt [hunk 2]
```diff
@@ -92,8 +92,8 @@ add_library(${TARGET_NAME} SHARED
 target_link_libraries(${TARGET_NAME}
   ${QT_LIBRARIES}
   dtkCore
-  dtkLog
-  ITKCommon
+  dtkLog  
+  ${ITK_LIBRARIES}
```
Comment (ocommowi) [inline src/plugins/legacy/medSegmentation/CMakeLists.txt:96]: Same as above why using ITK_LIBRARIES ?

File: superbuild/projects_modules/LogDemons.cmake [hunk 3]
```diff
@@ -39,8 +39,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Set up versioning control.
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/LCC-LogDemons.git)
-set(git_tag master)
+set(git_url ${GITHUB_PREFIX}medInria/LCC-LogDemons.git)
```
Comment (ocommowi) [inline superbuild/projects_modules/LogDemons.cmake:42]: Will need to be changed back to Inria-Asclepios master once all branches are validated and merged

File: superbuild/projects_modules/RPI.cmake [hunk 4]
```diff
@@ -40,8 +40,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Set up versioning control.
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/RPI.git)
-set(git_tag ITK5.0)
+set(git_url ${GITHUB_PREFIX}medInria/RPI.git)
```
Comment (ocommowi) [inline superbuild/projects_modules/RPI.cmake:43]: Same here, back to inria asclepios once everything merged

File: superbuild/projects_modules/TTK.cmake [hunk 5]
```diff
@@ -39,8 +39,8 @@ if (NOT USE_SYSTEM_${ep})
 ## Set up versioning control.
 ## #############################################################################
 
-set(git_url ${GITHUB_PREFIX}Inria-Asclepios/TTK-Public.git)
```
Comment (ocommowi) [inline superbuild/projects_modules/TTK.cmake:42]: For now, let's keep the architecture of repos. Once everything merged, move back to inria asclepios master

**Review by ocommowi — COMMENTED** [review]
Code looks good. I'll try a compilation from scratch as soon as I can

**Comment by ocommowi** [comment]
Everything now compiles (with a small correction added in inrimage io).

**Comment by ocommowi** [comment]
Hmm t'a pas oublié de prendre en compte mes commentaires de review ?

**Comment by Florent2305** [comment]
> Hmm t'a pas oublié de prendre en compte mes commentaires de review ?

Ha si carrément j'avais pas vu.
Bon ceci dit je pensais passer l'intégration continue dessus tel quel et faire une 2nd PR pour mettre des liens vers inria-asclepios.
Je comptai voir cela à notre réunion de demain pour savoir pour chaques projets si on garde des branches ou si on fusionne dans leurs masters respectif.

Pour le reste je te fais une réponse dans cette PR même si elle est plus "Open" ci cela te convient bien

**Comment by ocommowi** [comment]
Au passage, vu que t’es pas sur discord aujourd’hui, ca te va si on fait la reunion de main plutot dans l’aprem genre vers 16h ? J’ai un truc a aller faire en ville demain juste apres le seminaire…

Oliv

File: src/plugins/legacy/itkFilters/CMakeLists.txt (hunk 1 above)
Comment (Florent2305) [reply src/plugins/legacy/itkFilters/CMakeLists.txt:77]: Normally: When you specify a list of component on find_package the ${XXX_LIBRARIES} contain only needed  libraries.
I'm agree with you for the moment I still to prefer manually indicate all libraries and I wait medInria4.0 to investigate evaluate the interest to use  ${XXX_LIBRARIES}.
But for some linkerrors I couldn't determine which lib is needed. So I exceptionally used  ${XXX_LIBRARIES} and I don't generalize this process.

For  find_package without component list,  ${XXX_LIBRARIES} is still totally forbidden for me.

File: src/plugins/legacy/medSegmentation/CMakeLists.txt (hunk 2 above)
Comment (Florent2305) [reply src/plugins/legacy/medSegmentation/CMakeLists.txt:96]: Normally: When you specify a list of component on find_package the ${XXX_LIBRARIES} contain only needed  libraries.
I'm agree with you for the moment I still to prefer manually indicate all libraries and I wait medInria4.0 to investigate evaluate the interest to use  ${XXX_LIBRARIES}.
But for some linkerrors I couldn't determine which lib is needed. So I exceptionally used  ${XXX_LIBRARIES} and I don't generalize this process.

For  find_package without component list,  ${XXX_LIBRARIES} is still totally forbidden for me.

**Pull request merged**